tests/testthat/test-sclr.R

```diff
@@ -31,6 +31,14 @@ test_that("as.list strips attributes apart from names", {
   expect_equal(attributes(y), list(names = names(x)))
 })
 
+test_that("as.data.frame works", {
```

Comment:
I'm not 100% certain this is correct behaviour. Can you please file an issue instead of making a test?

Reply:
Done.